txScraper.py

```diff
@@ -24,7 +24,7 @@
 for row in table.findAll("tr"):
     for col in row.findAll("td"):
         for countyLink in col.findAll("a", href = True):
-            url = 'http://comptroller.texas.gov/propertytax/administration/pvs/findings/2013f/{}'.format(countyLink['href'])
+            url = 'https://comptroller.texas.gov/propertytax/administration/pvs/findings/2013f/{}'.format(countyLink['href'])
```

Comment:
Did you test this?

I looked very quickly at these URLs, and it seems they are all redirecting to https://www.comptroller.texas.gov/taxes/property-tax/ .  So probably no benefit pointing them at a HTTPS server, instead of HTTP.  This is probably fixing the wrong bug.